src/translations.js

```diff
@@ -26,16 +26,20 @@ module.exports = class Translation {
   getTranslationByKeyAndLocale (key, dependentKeys, locale, cb) {
     let self = this;
     locale = locale.toLowerCase();
-
+    locale = locale == "ch" ? "de" : locale;
     // If cache can be used, use it to serve translations;
     if (self.canUseCache(locale)) {
       return cb(null, self.fetchKeyFromJSON(key, dependentKeys, locale));
     } else {
       // if cache cannot be used , serve it from previous cache and update it in background.
       if (!_.isEmpty(self.translation[locale].value)) {
-        let cembraLocales=["ch","it","fr","en"];
+        let cembraLocales=["de","it","fr","en"];
```

Comment:
here also please confirm like above.

Reply:
yes, the downloaded file has the name "de.json" so i am saving it in cache based on file name as  "de" .